pkg/rid/store/store.go

```diff
@@ -14,6 +14,9 @@ type Store interface {
 
 	// Close closes the store and releases all of its resources.
 	Close() error
+
+	// Get store version
+	GetVersion(ctx context.Context) (string, error)
```

Comment:
I will update this to return semver.Version

Reply:
Sounds good, but that can be a follow-up PR.